src/app/dataset-block/metadata-block/components/event-details/components/set-polling-source-event/set-polling-source-event.component.html

```diff
@@ -0,0 +1,202 @@
+<div class="d-flex justify-content-end">
+    <span class="me-2">Show as yaml</span>
+    <app-toggle
+        [isToggled]="isYamlView"
+        (toggled)="onToggleView($event)"
+    ></app-toggle>
+</div>
+
+<div *ngIf="isYamlView" class="layout lh-1 font-monospace mt-4">
```

Comment:
We need to think of something more universal. There would be many kinds of events, and implementing the YAML view directly each time would be annoying. Instead, what I'd like to see is a universal component that can show some YAML structures based on the model.

So:
 - some object model describing YAML sections (something simple, tree-like structure)
 - some universal visual component (or components) that can render this model
 - a builder of the YAML model, that would produce it based on the input event structures, and depending on their types/data